pkgs/applications/misc/owmods-cli/default.nix

```diff
@@ -0,0 +1,63 @@
+{ lib
+, pkg-config
+, openssl
+, libsoup
+, fetchFromGitHub
+, installShellFiles
+, rustPlatform }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "owmods-cli";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "Bwc9876";
+    repo = "ow-mod-man";
+    rev = "cli_v${version}";
+    sparseCheckout = [
+      "owmods_cli"
+      "owmods_core"
+      "xtask"
+    ];
+    sha256 = "FtfGn6YzjwLn5gGOuG2nKR93wmR4wUYNou3oZiVG+zY=";
+  };
+
+  cargoLock = {
+    lockFile = ./Cargo.lock;
+    outputHashes = {
+      "tauri-plugin-fs-watch-0.1.0" = "sha256-Ei0j7UNzsK45c8fEV8Yw3pyf4oSG5EYgLB4BRfafq6A=";
+    };
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+    installShellFiles
+  ];
+
+  buildInputs = [
+    openssl
+    libsoup
+  ];
+
+  buildAndTestSubdir = "owmods_cli";
+
+  postInstall = ''
+    cargo xtask dist_cli
+    installManPage man/man*/*
+    installShellCompletion --cmd owmods \
+    dist/cli/completions/owmods.{bash,fish,zsh}
+  '';
+
+  meta = with lib; {
+    description = "CLI version of the mod manager for Outer Wilds Mod Loader";
+    longDescription = ''
+    The CLI interface for the mod manager for the Outer Wilds Mod Loader, made with clap.
```

Comment:
Personally, I think this longDescription is a little superfluous, as it doesn't add much.

Reply:
Removed the longDescription as suggested.